src/main/webapp/lectures.js

```diff
@@ -1,3 +1,7 @@
+/* Used to redirect users to the lecture view page, while storing lecture key in
+ * the URL. */
+const LINK_PREFIX = '/lecture-view.html?key=';
```

Comment:
I'd recommend using `URL.searchParams` to construct this.  For instance:
```
  const url = new URL('/lecture-view.html', window.location.origin);
  url.searchParams.append('key', key);
```

Reply:
fixed